source/_integrations/matter.markdown

```diff
@@ -107,18 +104,29 @@ Note: The section below mentions third-party Thread border routers such as the N
     - If you are already running the Matter server in another add-on, in or a custom container:
       - Deselect the checkbox, then select **Submit**.
       - In the next step, provide the URL to your Matter server.
+
+#### Check logos on the device
+
+- On the device packaging, check for both the Matter logo and for either the Wi-Fi or the {% term Thread %} logo.
+- Check if the QR code is only on the packaging or if it is also on the device.
+  - If it is only on the packaging, snap a picture of the QR code and the device and store the image and the numerical code in a save place.
+  - If you lose the QR code and disconnect the device at some point, you won't be able to connect to that device again without the QR code.
```

Comment:
From the device I've seen, most of them are on the device (at least Eve, Onvis plug, Nanoleaf). But some have in their documentation (Aqara  Door and Window Sensor P2 is an example).

I'd reword this a bit to:

```suggestion
- Check if the QR code or the numeric setup code is on the device.
  - If you reset your device you'll need the QR code *or* numeric setup code to {% term commission %} that device again! Without this information, commissioning won't be possible!
  - It is good practice to snap a picture of the QR code and/or numeric setup code as a backup, ideally along with the device for reference, and store the image in a save place.
```